src/CsrfGuard.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Linna;
 
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\Exception\UnsatisfiedDependencyException;
```

Comment:
Hi! Thanks for enhancements, these lines of code seems unused inside the class!

Reply:
Yes, I forgot to remove that namespace after finishing my experiment.

Reply:
Ok 👍